apps/frontend/src/pages/landing/components/canvas-step/CanvasStep.tsx

```
@@ -0,0 +1,672 @@
+                    </button>
+                    <button
+                      onClick={() => setSelectedTool('hand')}
+                      className={cn(
+                        'p-3 rounded-full transition-all',
+                        selectedTool === 'hand' ? 'bg-primary text-white' : 'bg-gray-100 text-gray hover:bg-primary-bg',
+                      )}
+                      title="이동 커서"
+                    >
+                      <HandBackRightIcon className="size-4" />
+                    </button>
+                    <button
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)